packages/material-tree-renderer/src/services/property.util.ts

```diff
@@ -56,7 +53,7 @@ export const findPropertyLabel = (property: Property): string => {
     [
       property.schema.title,
       property.label,
-      property.schema.$id,
+      (property.schema as any).$id || (property.schema as any).id,
```

Comment:
Fix casts

Reply:
Since using `id` is not mandatory, should we really check this?